frontend/src/services/http/HttpResponseInterceptors.ts

```diff
@@ -13,10 +14,14 @@ const MAX_RETRY = 2
 // --- Interceptors
 
 // Assume it is caught in error block based on default handling.
-async function forbiddenStatusItcpt(error: any) {
+async function accessErrorResponsesItcpt(error: any) {
+    if (error.response.status == 401 && (retryCount.value < MAX_RETRY)) {
+        return refreshTokenAndReTry(error.request.responseURL, error.response);
```

Comment:
Hello Nick,
I think I know why 'error.request' not working before now.
Could you please change this `error.request` to `error.config`. It should be "**.config**"
Please give it a test. Sorry for more changes.
Thank you Nick, I will try to approve this tomorrow quick.

Reply:
Nice find! It does work with ".config".